app/editor/src/store/hooks/storage/useStorage.ts

```diff
@@ -15,6 +15,7 @@ interface IStorageController {
   ) => Promise<IItemModel>;
   download: (locationId: number, path: string) => Promise<unknown>;
   stream: (locationId: number, path: string) => Promise<unknown>;
+  stream2: (locationId: number, path: string) => Promise<string>;
```

Comment:
Why do we now have two stream endpoints in this hook?

Reply:
Updated, initially thought to keep the original functionality for other potential scenario. Thanks